src/main/java/org/gridsuite/modification/server/entities/equipment/modification/SubstationFreePropertyEntity.java

```diff
@@ -0,0 +1,46 @@
+/**
+ *  Copyright (c) 2023, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.server.entities.equipment.modification;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+@Getter
+@NoArgsConstructor
+@SuperBuilder
+@Entity
+@Table(name = "substation_free_property")
+public class SubstationFreePropertyEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "name", nullable = false)
+    String name;
```

Comment:
add private to the other class attributes for code consistency
```suggestion
    private String name;
```